README.md

```diff
@@ -1,12 +1,25 @@
-## Get started
+# 📃⏳ Achill
+Web app to help you fill out Troi. Uses the [Svelte](https://svelte.dev/) framework and the [Troi API](https://v2.troi.dev/).
+
+#### 👉 Also check out our other in-house self-built Slack bot for Troi: [BleibTroy](https://github.com/digitalservice4germany/troi-slack-bot)
```

Comment:
Amazing! :D "Bleibtroy". 😂

Reply:
@benjaminaaron To fix the build, do you maybe wanna run `npm run format`?

Reply:
Hehe 😄 
Done ✅